.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
+OPENAI_API_KEY=
```

Comment:
Consider adding a check to ensure these environment variables are set before running the application. This helps in identifying issues early during startup rather than at runtime.